.github/actions/custom-actions/aselo_development_custom/action.yml

```diff
@@ -254,4 +230,9 @@ runs:
 
     - name: Add MODICA_TEST_SEND_MESSAGE_URL
       run: echo "MODICA_TEST_SEND_MESSAGE_URL=${{ env.MODICA_TEST_SEND_MESSAGE_URL }}" >> .env
+      shell: bash
+
+
+    - name: Add DELEGATE_WEBHOOK_URL
+      run: echo "DELEGATE_WEBHOOK_URL=https://hrm-development.tl.techmatters.org/lambda/twilio/account-scoped" >> .env
```

Comment:
Shouldn't this be in the "general" env variables, as part of the main action? I'm asking cause I assume this URL will be required for every account, not just some of them (this file intended to be used in the later).

Reply:
We probably will when we roll it out everywhere, I just wanted to be extra cautious about where we switch on the delegation code until we're sure it's working ok, since it might completely break our task router handling if it's broken